cashier-paddle.md

```diff
@@ -0,0 +1,861 @@
+
+For more information, you may review [the Paddle API documentation on pay link generation](https://developer.paddle.com/api-reference/product-api/pay-links/createpaylink).
+
+<a name="inline-checkout"></a>
+### Inline Checkout
+
+If you don't want to make use of the "overlay" style checkout widget, Paddle also has an option to display the widget inline. While this approach does not allow you to adjust any of the checkout's HTML fields, it allows you to embed the widget within your application.
+
+To make it easy for you to get started with inline checkout, Cashier includes a `paddle-checkout` Blade component. To get started, you should [generate a pay link](#pay-links) and pass the pay link to the component's `override` attribute:
+
+    <x-paddle-checkout :override="$payLink" class="w-full" />
+
+Since `height` is a preset value for inline checkouts you'll need to pass it explicitly as an attribute if you want to adjust it:
+
+    <x-paddle-checkout :override="$payLink" class="w-full" height="500" />
+
+Alternatively, you can fully customize the widget with custom options instead of using a pay link:
+
+    $options = [
+        'product' => $productId,
+        'title' => 'Product Title',
+    ];
+
+Then pass the options to the Blade component:
+
+    <x-paddle-checkout :options="$options" class="w-full" />
+
+Please consult Paddle's [guide on Inline Checkout](https://developer.paddle.com/guides/how-tos/checkout/inline-checkout) as well as their [Parameter Reference](https://developer.paddle.com/reference/paddle-js/parameters) for further details.
+
+<a name="user-identification"></a>
+### User Identification
+
+In contrast to Stripe, Paddle users are unique across the whole of Paddle, not unique per Paddle account. Because of this, Paddle's API's do not currently provide a method to update a user's details such as their email address. When generating pay links, Paddle identifies users using the `customer_email` parameter. When creating a subscription, Paddle will try to match the user provided email to an existing Paddle user.
+
+In light of this behavior, there are some important things to keep in mind when using Cashier and Paddle. First, when a new subscription is created we will save the `customer_email` value in the database as the user's `paddle_email` column value. **It is extremely important that you do not modify this value.** Cashier will use the `paddle_email` value for every new subscription and [pay link](#pay-links) to ensure all transactions are linked to the same customer within Paddle. Furthermore, this will ensure that you can assign multiple Paddle subscriptions to a single user using Cashier. A caveat of this is that any [override of the `customerEmail` method](#customer-defaults) on a billable user won't work anymore after the initial subscription has been created.
+
```

Comment:
What do you mean by "override of customerEmail method on a billable user won't work anymore"?

Reply:
In the section of "Customer Defaults" I explain that you can set a couple of methods that'll pre-fill the checkout widget for a customer so they can immediately move to the payment information part.

What I explain above is that that default will be ignored after the first subscription has been created. It's vital that the original email address on the user is used to create any subsequent subscriptions and payments so they're all attached to the same user within Paddle.